Assets/Scripts/CameraFollow.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraFollow : MonoBehaviour
+{
+    [SerializeField] private GameObject target;
+    [SerializeField] private float DampTime;
+
+    private void FixedUpdate()
```

Comment:
This code should be done in the late Update not fixed update as late update runs in the last after fixed update and update.So Camera movement should be done in the late update part.